src/rocksdb.rs

```diff
@@ -1086,7 +1086,7 @@ fn iterator_test() {
 
 #[test]
 fn non_unicode_path_test() {
-    let path = "путь_не_юникод/_rust_rocksdb_unicode_test";
+    let path = "ÇéæåÑëê/_rust_rocksdb_unicode_test";
```

Comment:
I must be misunderstanding something. Isn't this using unicode chars? The test name seems to imply that it checks that the code works with paths encoded with non-unicode? Shouldn't this use something like `encoding_rs` to produce a non-unicode string and use that as the path?

Also: after running tests there's a `ÇéæåÑëê/` folder in the root. Perhaps the test should clean up?

Reply:
This test failed on Windows because it doesn't use unicode and the default code page (which is used on the CI) doesn't contain Cyrillic characters. I'll add the cleanup code to remove the folders.

Reply:
> failed on Windows because it doesn't use unicode

Then perhaps the test name should change here? It says "**non**_unicode_path_test", so a path with a different encoding should work but on windows it doesn't (uses UTF-32 right? which is still unicode...).

> the default code page (which is used on the CI)

Is CI running Linux? And the code page used by the shell running the test shouldn't stop rust or the OS from creating a unicode path, or am I assuming the wrong thing?

I'm confused!

Reply:
I've renamed the test to `non_ascii_path_test` since that's what we're testing.
Rust uses UTF-8 strings so on Linux / macOS there's nothing to do. Windows doesn't unicode though, it uses ANSI encoding which has an associated code page. By default, [this](https://en.wikipedia.org/wiki/Windows-1252) codepage is used. So we need to convert all the rust strings to/from ANSI encoding using the `local_encoding` crate. This PR adds an AppVeyor build which makes CI run on Windows as well, and there using the default code page the Cyrillic characters were not available and the test would fail.

Reply:
Ok, so it's essentially a windows limitation. Maybe we should have two tests, one for "non ascii latin1 chars` that runs on all platforms and one "unicode outside latin1" that runs one non-windows platforms using cyrillic/japanese.

But it's a minor issue. Thanks for explaining.

Reply:
I'm pretty sure that the test was implemented by one of our friends with a Cyrillic keyboard :wink: And probably tested on Windows as well using a code page with Cyrillic symbols. But since I added the AppVeyor build it showed that the test was dependent on the Windows environment and I needed to change it.